hikari/impl/entity_factory.py

```diff
@@ -3388,6 +3388,7 @@ def deserialize_template(self, payload: data_binding.JSONObject) -> template_mod
             updated_at=time.iso8601_datetime_string_to_datetime(payload["updated_at"]),
             source_guild=source_guild,
             is_unsynced=bool(payload["is_dirty"]),
+            app=self._app,
```

Comment:
Maybe move it as the first argument for the sake of consistency :)

Reply:
Done 💞